xc7/archs/artix7/tiles/iopad/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+project_xray_tile(
+  PART artix7
```

Comment:
I don't imagine this actually works?

Reply:
No, not yet. Do I need to define pb_type/model.xml for all SITE_TYPES? It seems that way.